frontend/public/components/utils/name-value-editor.jsx

```diff
@@ -351,11 +351,11 @@ const PairElement = DragSource(DRAGGABLE_TYPE.ENV_ROW, pairSource, collectSource
           }
           {
             !readOnly &&
-              <div className="col-xs-1">
+              (index > 0 && (<div className="col-xs-1">
```

Comment:
This doesn't look right.... The remove button will clear the value even if there's only one. Otherwise you have to manual clear both inputs, which is a lot more work for the user.